### PR TITLE
Site: add icon to side bar items

### PR DESCRIPTION
**Pull request by coolamiy**

### CODE REVIEW

**Comment by coolamiy** [comment]
preview link https://deploy-preview-12040--kubernetes-sigs-minikube.netlify.app/docs/

**Comment by medyagh** [comment]
@coolamiy  thank you for this PR

could we change the Green Icon to something black maybe a Square ? or something more in harmony ?

or would there be a font-awesome for each item ? like FAQ or Presentation....(maybe that is impossibel to have unique icon for each ...)

**Comment by medyagh** [comment]
also for other items like Benchmark we should have the same -> ...since benchmark also has sub items

**Comment by coolamiy** [comment]
> @coolamiy thank you for this PR
> 
> could we change the Green Icon to something black maybe a Square ? or something more in harmony ?
yes ..  can you please pick one from the following ?
https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=square

> 
> or would there be a font-awesome for each item ? like FAQ or Presentation....(maybe that is impossibel to have unique icon for each ...) we can achieve by adding custom icon section in go templating  but downside would be if a new section is added that needs to be updated in the template as well for each parent level logo 
if thats the way to go please provide font-awesome icon from the list 
https://fontawesome.com/v5.15/icons?d=gallery&p=2

Note: icons i was using is unicode icon and does not depend on font-awesome at all.

**Comment by coolamiy** [comment]
> also for other items like Benchmark we should have the same ➔ ...since benchmark also has sub items

i will fix that .. thanks

File: site/layouts/partials/sidebar-tree.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
   <ul class="td-sidebar-nav__section pr-md-3">
     <li class="td-sidebar-nav__section-title">
       <!-- Link for subections -->
-      <a href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $expandSection }} collapsed{{ end }}{{ if $activeSection}} active{{ end }} td-sidebar-link td-sidebar-link__section">{{ $s.LinkTitle }}</a>
+      <a href="{{ $s.RelPermalink }}" class="align-left pl-0 pr-2{{ if not $expandSection }} collapsed{{ end }}{{ if $activeSection}} active{{ end }} td-sidebar-link td-sidebar-link__section">🍀 {{ $s.LinkTitle }}</a>
```
Comment (medyagh) [inline site/layouts/partials/sidebar-tree.html:71]: how about using this character  ▀ instead ?

File: site/layouts/partials/sidebar-tree.html [hunk 2]
```diff
@@ -100,7 +100,7 @@
           {{ end }}
 
           {{ if $showPage }}
-            <a class="td-sidebar-link td-sidebar-link__page {{ if $activePage }} active{{ end }}" id="{{ $mid }}" href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>
+            <a class="td-sidebar-link td-sidebar-link__page {{ if $activePage }} active{{ end }}" id="{{ $mid }}" href="{{ .RelPermalink }}"> ➔ {{ .LinkTitle }}</a>
```
Comment (medyagh) [inline site/layouts/partials/sidebar-tree.html:103]: how about we add a blank space before the " ➔" so it indents more ?

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/layouts/partials/sidebar-tree.html (hunk 2 above)
Comment (medyagh) [reply site/layouts/partials/sidebar-tree.html:103]: or how about we leave the Sub Item ones Without any icons. just the Parent ones have a  ▀  and these sub items should have an Extra Blank Space so it indents more....could we try that first ?

Comment (coolamiy) [reply site/layouts/partials/sidebar-tree.html:103]: updated preview link.
https://deploy-preview-12040--kubernetes-sigs-minikube.netlify.app/docs/

File: site/layouts/partials/sidebar-tree.html (hunk 1 above)
Comment (coolamiy) [reply site/layouts/partials/sidebar-tree.html]: used &#11035; (solid black square )

**Comment by coolamiy** [comment]
#12073

**Comment by coolamiy** [comment]
#12073

**Comment by medyagh** [comment]
@coolamiy that is really nice ! I love that the selected item becomes blue

I am thinking how about we choose a different shape, how about this bullet 

https://www.alt-codes.net/bullet_alt_codes.php

maybe empty bullet and filled bullet for selected, could we please try that too?

**Comment by medyagh** [comment]
For some reason the netlify error failed

**Comment by medyagh** [comment]
@coolamiy do u mind pulling upstream to trigger the netlify ?

**Comment by coolamiy** [comment]
updated, Please review.

**Comment by medyagh** [comment]
Looks great thank you very much !

**Pull request merged**